src/utils/db.js

```diff
@@ -0,0 +1,17 @@
+const { Sequelize } = require('sequelize');
+const utils = require('util');
+
+global.TextEncoder = utils.TextEncoder;
```

Comment:
The assignment `global.TextEncoder = utils.TextEncoder;` is unnecessary and could potentially cause issues. The `TextEncoder` is natively available in Node.js environments, so this line can be removed unless there is a specific reason for this assignment.